test/e2e/network/loadbalancer.go

```diff
@@ -141,13 +133,7 @@ var _ = common.SIGDescribe("LoadBalancers", func() {
 	})
 
 	f.It("should be able to change the type and ports of a TCP service", f.WithSlow(), func(ctx context.Context) {
-		// requires cloud load-balancer support
-		e2eskipper.SkipUnlessProviderIs("gce", "gke", "aws")
-
 		loadBalancerLagTimeout := e2eservice.LoadBalancerLagTimeoutDefault
-		if framework.ProviderIs("aws") {
-			loadBalancerLagTimeout = e2eservice.LoadBalancerLagTimeoutAWS
```

Comment:
hm... I guess we should probably use the larger value (TimeoutAWS) rather than the smaller one